autoparallel/propagation_rules.py

```diff
@@ -646,31 +646,6 @@ def convert_element_type_rule(mesh, op_schema):
     return out_strat
 
 
-@register_opschema_rule(torch.ops.aten.split.Tensor)
-def split_rule(mesh, op_schema):
```

Comment:
`aten.split.Tensor` has been implemented upstream in https://github.com/pytorch/pytorch/pull/149106